react/features/connection-indicator/components/ConnectionIndicator.js

```diff
@@ -1,48 +1,57 @@
+import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 
+import { translate } from '../../base/i18n';
 import { JitsiParticipantConnectionStatus } from '../../base/lib-jitsi-meet';
 import { Popover } from '../../base/popover';
 import { ConnectionStatsTable } from '../../connection-stats';
 
 import statsEmitter from '../statsEmitter';
 
-declare var $: Object;
 declare var interfaceConfig: Object;
 
-// Converts the percent for connection quality into a string recognized for CSS.
+/**
+ * The connection quality percentage that must be reached to be considered of
+ * good quality and can result in the connection indicator being hidden.
+ *
+ * @type {number}
+ */
+const INDICATOR_DISPLAY_THRESHOLD = 70;
+
+/**
+ * An array of display configurations for the connection indicator and its bars.
+ * The ordering is done specifically for faster iteration to find a matching
+ * configuration to the current connection strength percentage.
+ *
+ * @type {Object[]}
+ */
 const QUALITY_TO_WIDTH = [
 
-    // Full (5 bars)
+    // Full (3 bars)
     {
-        percent: 80,
+        colorClass: 'status-high',
+        percent: INDICATOR_DISPLAY_THRESHOLD,
```

Comment:
Should we use this constant here? I don't think that logically it is related to the `QUALITY_TO_WIDTH` array.

Reply:
The constant does not have to be used but the values are related. A good connection is at 70% or higher connectivity. (The number 70 is something I chose arbitrarily and likely will get adjusted.) Three bars should be displayed when there is a good connection. Maybe if QUALITY_TO_WIDTH were renamed it would feel better.